tmp/build-errors.log

```diff
@@ -0,0 +1 @@
+exit status 1exit status 1exit status 1exit status 1exit status 1
```

Comment:
i am not sure we need to commit the tmp dir at all?

Reply:
This PR is closed.